app/controllers/sign_up/completions_controller.rb

```diff
@@ -116,6 +116,7 @@ def email_domain
     def track_completion_event(last_page)
       analytics.user_registration_complete(**analytics_attributes(last_page))
       user_session.delete(:in_account_creation_flow)
+      user_session.delete(:pending_completions_consent)
```

Comment:
We don't have this session value anymore.

```suggestion
```

Reply:
Right! That is now removed.